processor/eqreceipt/app.go

```diff
@@ -0,0 +1,124 @@
+package eqreceipt
```

Comment:
Again, with the package structure, nesting this package under processor seems unnecessary, unless you're envisaging multiple potential processors? At which point you'd probably want to think about restructuring the App to be something that could be shared across all processors, and then specific implementations could live nested or in their own pacakges - you'd probably need some 'Producer' and 'Consumer' interfaces in your App to allow that level of flexiblity though.

As it stands I think you just don't need to nest this as far as you have, just processor or eqreceipt would be fine on their own